### ISSUE TITLE
"No issues found" message if no checks return any errors/warnings

### ISSUE THREAD

**Issue by fgblomqvist** [fi3ework/vite-plugin-checker#58]
If I have an ESLint error or two, then fix them, I see nothing in the console that indicates that I have done so.

On top of that, the Typescript checker currently seems to emit its own message ("Found X errors. Watching for file changes"). However it's not clear that it's coming from TS.

Instead of the above, I propose this:
* If all checkers run and there are no issues with any of them, print a pretty "No issues found, waiting for file changes..." message (maybe in green or something cool, maybe with a checkmark, idk)
* Mute TypeScripts own message

OR

* If a checker runs and there are no issues, print a pretty "[ESLint] No issues found, waiting for file changes..." message (so one for each checker)
* Mute TypeScripts own message

The first solution is preferred I think, but if everything is running async the latter might be better.
In short, it should be clear when the checker(s) run and there were no issues, just to clarify that if there were previously issues, there are none anymore.

**Comment by fi3ework** [comment]
fixed in #61

**Comment by fgblomqvist** [comment]
This wasn't fixed in #61, it needs more work. E.g. the outputting of another message etc.

**Comment by christopher-caldwell** [comment]
`Found 0 errors. Watching for file changes.` is the standard error message from the TSC in watch mode. 

When there is an error, the message is in red, and very easy to see. 
> ERROR(TypeScript)  Property 'sa' does not exist on type 'PaletteColor'

Perhaps the all clear could be green? Prefix could be `INFO`, `MESSAGE`, `SUCCESS`, etc
> INFO(TypeScript) Found 0 errors. Watching for file changes

**Comment by fi3ework** [comment]
implemented in 0.4.2. ESLint and VLS check summary are added.
